lib/mpi/models/mvi_profile_ids.rb

```diff
@@ -21,13 +21,18 @@ module MviProfileIds
       attribute :birls_id, String
       attribute :birls_ids, Array[String]
       attribute :sec_id, String
+      attribute :sec_ids, Array[String]
       attribute :vet360_id, String
       attribute :cerner_facility_ids, Array[String]
       attribute :cerner_id, String
 
       def mhv_correlation_id
         @active_mhv_ids&.first
       end
+
+      def multiple_sec_ids?
```

Comment:
I don't think the presence of multiple `sec_ids` is relevant for the mvi profile parser, it's only really relevant for the actual authentication process. So I probably wouldn't put this function here and would only keep it in the actual auth checks